phpstan.neon

```diff
@@ -1,22 +1,42 @@
+        - '#Parameter .+ of function ucfirst expects string, string\|null given\.#'
+        - '#Parameter .+ of function strpos expects string, string\|null given\.#'
+        - '#Parameter .+ of function base64_decode expects string, string\|null given\.#'
+        - '#Parameter .+ of function md5 expects string, string\|null given\.#'
+        - '#Parameter .+ of method .+ expects Orm\\Zed\\Payone\\Persistence\\SpyPaymentPayone, Orm\\Zed\\Payone\\Persistence\\SpyPaymentPayone\|null given\.#'
+        - '#Parameter .+ of method .+ expects .+Interface, .+AbstractRequestContainer given\.#'
+        - '#Parameter .+ of method .+ expects .+AbstractRequestContainer, .+Interface given\.#'
+        - '#Parameter .+ of method .+ expects .+Interface, .+Interface given\.#'
+        - '#Parameter .+ of method .+ expects .+AbstractPaymentMethodContainer, .+ContainerInterface given\.#'
+        - '#Parameter .+ of method .+ expects Generated\\Shared\\Transfer\\QuoteTransfer, Generated\\Shared\\Transfer\\QuoteTransfer\|null given\.#'
+        - '#Property SprykerEco\\Zed\\Payone\\Business\\Api\\Call\\CreditCardCheck::\$storeCardData is never read, only written\.#'
+        - '#Parameter .+ of method .+ expects Generated\\Shared\\Transfer\\SaveOrderTransfer, Generated\\Shared\\Transfer\\SaveOrderTransfer\|null given\.#'
+        - '#Parameter .+ of method .+ expects Generated\\Shared\\Transfer\\PayonePaymentTransfer, Generated\\Shared\\Transfer\\PayonePaymentTransfer\|null given\.#'
+        - '#Parameter .+ of method .+ expects Generated\\Shared\\Transfer\\PaymentDetailTransfer, Generated\\Shared\\Transfer\\PaymentDetailTransfer\|null given\.#'
+        - '#Call to method .+ on an unknown class SprykerEco\\Zed\\Payone\\Business\\Payment\\RequestSender\\CaptureContainerInterface\.#'
+        - '#Parameter \$requestContainer of method .+mapProductData\(\) has invalid type .+ContainerInterface\.#'
+        - '#Parameter \$container of method .+preparePartialCaptureOrderItems\(\) has invalid type .+ContainerInterface\.#'
         - '#Call to method .+ on an unknown class Generated\\.+\\Ide\\AutoCompletion\.#'
-        - '#Property .+ has unknown class Orm\\Zed\\Payone\\Persistence\\.+ as its type\.#'
-        - '#Call to an undefined method Spryker\\Shared\\Kernel\\Transfer\\AbstractTransfer::.+#'
         - '#Argument of an invalid type .+ supplied for foreach, only iterables are supported\.#'
-        - '#Parameter .+ .+ of method .+ expects SprykerEco\\Zed\\Payone\\Business\\.+, SprykerEco\\Zed\\Payone\\Business\\Api\\Request\\Container\\.+ given\.#'
-        - '#Method SprykerEco\\Zed\\Payone\\Business\\Payment\\MethodMapper\\.+ should return SprykerEco\\Zed\\Payone\\Business\\Api\\Request\\Container\\.+ but returns SprykerEco\\Zed\\Payone\\Business\\Api\\Request\\Container\\.+#'
+        - '#Method .+ should return .+ but returns .+\.#'
+        - '#Call to an undefined method .+\.#'
     excludePaths:
```

Comment:
Why have you added all those to ignore? Imho these should be fixed instead ...

Reply:
It will take a huge amount of time. Maybe, we will return to this point in the future. Now we must release this PR as soon as possible.

Reply:
But I agree, having them excluded can open door for hidden Bugs.

Reply:
It's not a half. We had more than 300 errors at the beginning. The most were fixed. Only controversial points were remain (or points for deep refactoring). I agree, that they must be addressed again. We can create new task for refactoring and make it immediately after release this huge PR. We discussed it many times with @dereuromark, this PR is already overloaded, let's finally release it.